erpnext/hr/doctype/hr_settings/hr_settings.json

```diff
@@ -143,13 +144,19 @@
    "fieldname": "send_leave_notification",
    "fieldtype": "Check",
    "label": "Send Leave Notification"
+  },
+  {
+   "default": "8",
+   "fieldname": "default_working_hours",
```

Comment:
I think **Standard Working Hours** will sound Better.

Add a checkbox **Fetch Standard Working Hours From Shift Type**.
if unchecked, Show this field to capture **Standard Working Hours**.